test/ecto/adapters/postgres_test.exs

```diff
@@ -715,6 +715,28 @@ defmodule Ecto.Adapters.PostgresTest do
     assert SQL.execute_ddl(drop) == ~s|DROP INDEX CONCURRENTLY "posts$main"|
   end
 
+  test "create check constraint, specifying constraint name" do
+    create = {:create, check_constraint(:products, check: "price > 0", name: "price_must_be_positive")}
+    assert(
+    SQL.execute_ddl(create) == ~s|ALTER TABLE "products" ADD CONSTRAINT "price_must_be_positive" CHECK (price > 0)|
+    )
+  end
+
+  test "create check constraint, not specifying constraint name" do
+    exception =
+      assert_raise(ArgumentError, ~r/supply a name/, fn ->
```

Comment:
No need to check this here. We check on Ecto.Migration tests. So you can erase this test.